cumulus/client/consensus/aura/src/collators/lookahead.rs

```diff
@@ -374,14 +380,17 @@ where
 				)
 				.await;
 
-				let allowed_pov_size = if cfg!(feature = "full-pov-size") {
+				let allowed_pov_size = if let Some(max_pov_size) = params.experimental_max_pov_size
+				{
+					max_pov_size
+				} else if cfg!(feature = "full-pov-size") {
```

Comment:
Hmm IMO we can just remove `full-pov-size` feature. Currently its not safe to use, and after this PR we will have 85% by default. So anyone who wants to live really risky can use the CLI to enable this.